queue/queue.py

```diff
@@ -3,13 +3,23 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
```

Comment:
Strongly consider using a LInkedList() as the storage structure for this as opposed to list/array.

The LInkedList() will allow us to have constant time operations for adding/removing items while the list/array will not.